pkg/cmd/template/cmd_test.go

```diff
@@ -210,6 +210,94 @@ data: #@ data.read("/funcs/data")`)
 	assert.Equal(t, expectedYAMLTplData, string(file.Bytes()))
 }
 
+func TestDataListRelativeToRootInDataValuesAndSchema(t *testing.T) {
```

Comment:
These tests give me _a lot_ of confidence that our "absolute path" loading works in both schema and data values -flavored templates. 👍🏻  Thank you for investing in this.

As I look around at the test files in `pkg/cmd/template` ... their ranks are growing. `cmd_test.go` is almost a KLOC.

If you haven't already thought about this, I'm wondering if these tests would be more "at home" in one of these test files:
- `cmd_root_library_test.go`
- `cmd_data_values_test.go`